src/main/java/org/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -251,7 +251,6 @@ public static void openConsole(File file) throws IOException {
                                                   Localization.lang("Error occured while executing the command \"%0\".", commandLoggingText),
                                                   Localization.lang("Open console") + " - " + Localization.lang("Error"),
                                                   JOptionPane.ERROR_MESSAGE);
-                    JabRefGUI.getMainFrame().getDialogService().notify(null);
                 }
```

Comment:
I just saw that we seemed to forgot to convert that old swing dialog here. Could you please fix this as well?
dialogService.showErrorDialog or so should work
Thanks!

Reply:
did it in f2a31d0